### PR TITLE
Resources section in documentation

### LINKED ISSUE #5763: Read me file with explanation to demo module
Demo module include allot of relevant and helpfull stuff, but the problem is that you need doing deep research to understand what is what (especially if you new to orchard core).
It will be nice to comment / add notes to that module (similar like Lombiq did to the demo they create)

Also, I think this module is amazing, and the video from YouTube really help to learn Orchard core , especially for the teams that moving from Joomla / WordPress.

Thanks!

### PR DESCRIPTION
**Pull request by agriffard**
Fixes #5763, #5775
Replaces #3955

### CODE REVIEW

**Comment by agriffard** [comment]
https://docs.orchardcore.net/en/learn/

**Comment by hishamco** [comment]
Just fixed the above link

**Comment by sebastienros** [comment]
I am suggesting "Resources" as "learn" is too generic, and covers all other sections already.

**Comment by agriffard** [comment]
Learn renamed into Resources

File: mkdocs.yml [hunk 1]
```diff
@@ -141,5 +143,6 @@ nav:
             - Tenants: docs/reference/modules/Tenants/README.md
             - Workflows: docs/reference/modules/Workflows/README.md
             - DataProtection (Azure Storage): docs/reference/modules/DataProtection.Azure/README.md
-    - Glossary: docs/glossary/README.md
-    
+    - Resources: 
+        - Learning: docs/resources/README.md
+
```
Comment (hishamco) [inline mkdocs.yml]: Remove extra line

**Pull request merged**